Cubical/Algebra/Polynomials/UnivariateList/UniversalProperty.agda

```diff
@@ -0,0 +1,8 @@
+{-# OPTIONS --safe #-}
+module Cubical.Algebra.Polynomials.UnivariateList.UniversalProperty where
```

Comment:
Is there a reason why you didn't want to prove the universal property here? That would resolve the issue about having a statement about Algebras in CommAlgebras.

Reply:
I think my reasoning was, that it would be good to be consistent about where things are. So 'Polynomials' could just contain a lot of pointers to Instances of CommRings and CommAlgebras and their Properties *and* theorems on the relations between different implementations of polynomials. But we are actually far away from that now, so I could move the ump as well.

Reply:
Fair enough, I hadn't looked at the `Polynomials/` too much but that seems to be more or less the standard, so let's try to keep it that way.

Reply:
A notable exception seems to be the UnivariateList polynomials, which have their ring structure defined in Polynomials.UniveriateList.Properties. I'll make an issue with the suggestion to move that.

Reply:
Done, #944